### DISCUSSION TITLE
Issue about creating organization

### DISCUSSION THREAD

**Discussion by abelshi** [community/community#125482] [npm]
Hi npm team,
I got an issue when I tried to create an organization 'lseg-analytics' under account 'analyticssdk'. It told me that 'The organization name lseg-analytics is not available'.

I think that is because, I used this organization name in another account 'anelshi', but after test in this account, I have already deleted this organization.

But I still cannot use this organization name in my new account 'analyticssdk'.

Can you please help me to check this issue? Our team really need this to publish our package.

Thanks
Abel

**Comment by karthickkumarsm** [comment]
It sounds like you've encountered an issue with the organization name 'lseg-analytics' being unavailable despite having deleted it from another account 'anelshi'. 

Here are a few steps you could take to troubleshoot this:

1. **Clear Cache and Cookies**: Sometimes, cached data can cause such issues. Try clearing your browser's cache and cookies, or even use a different browser altogether to see if the problem persists.

2. **Check for Partial Deletion**: Ensure that the organization 'lseg-analytics' was fully deleted from the 'anelshi' account. Sometimes, organizations might not be completely removed, causing the name to still be reserved.

3. **Contact Support**: If the issue persists, reach out to the support team of the platform you're using. They should be able to provide specific assistance and investigate any potential backend issues.

4. **Try a Variation of the Name**: If the organization name 'lseg-analytics' remains unavailable, consider using a slightly modified version of the name, if possible. For instance, appending numbers or changing the spelling slightly.

5. **Patience**: Sometimes, changes in account settings may take some time to propagate through the system. Waiting for a while before retrying might resolve the issue.

If none of these steps resolve the problem, the support team of the platform you're using would likely be your best resource for further assistance.

**Reply by grandmas-favorite** [reply]
Hi @karthickkumarsm , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by alfie80** [comment]
We are facing similar issue and wonder if anyone can help look to such problem.
